### ISSUE TITLE
Add role, aria, and schema attributes where appropriate

### ISSUE THREAD

**Issue by JacobDB** [JacobDB/new-site#29]
Adding role and aria attributes where appropriate would vastly improve accessibility on the sites that I work on, not to mention search engine optimization. This is a complex topic, so I won't try listing out everywhere this should be used. Instead, referring to the information below.

More information:
- http://stackoverflow.com/questions/10403138/what-is-the-purpose-of-the-role-attribute-in-html
- https://www.w3.org/TR/2007/WD-xhtml-role-20071004/
- http://schema.org/

**Comment by JacobDB** [comment]
I think this is in a good enough place for the time being.
